intro-HPC/ch_troubleshooting.tex

```diff
@@ -0,0 +1,58 @@
+\chapter{Troubleshooting}
+\label{ch:troubleshooting}
+
+
+
+\section{Walltime issues}
+If you get from your job output error a message similar to this:
+
+\begin{prompt}
+ =>> PBS: job killed: walltime 146 exceeded limit 120
```

Comment:
I would use `86400` (24h) rather than `120` (submitting such short jobs is unrealistic/bad practice)